files/zh-cn/web/api/htmlimageelement/srcset/index.html

```diff
@@ -7,49 +7,47 @@
+<li>要指定在某个宽度下使用特定图像候选代表的图像资源， 请提供 <strong>宽度描述符</strong>，包含表示该宽度的数字（以像素为单位）+ 小写字母“w”。例如，渲染一个450 像素宽的图像对应的描述符字符串： <code>450w</code>。指定的宽度必须是正数、非零整数，并且<em>必须</em>与引用图像的固有宽度相匹配。</li>
+<li>或者，您可以使用<strong>像素密度描述符</strong>，它指定了在什么样的显示器像素密度下应用相应的图像资源。它是通过将像素密度声明为正的非零浮点值，后跟小写字母“x”来编写的。例如，要指定在像素密度是标准密度的两倍时使用相应的图像，您可以提供像素密度描述符 <code>2x</code> 或 <code>2.0x</code>。< /li>
 
-<pre>"images/team-photo.jpg 1x, images/team-photo-retina.jpg 2x, images/team-photo-full 2048w"</pre>
+</ul>
 
-<p>This string provides versions of an image to be used at the standard pixel density (<code>1x</code>) as well as double that pixel density (<code>2x</code>). Also available is a version of the image for use at a width of 2048 pixels (<code>2048w</code>).</p>
+<p>您可以混合使用这两种类型的描述符。但是，您不能为同一个描述符指定多个图像候选字符。以下都是有效的图像候选字符串：</p>
+<pre>"images/team-photo.jpg 1x, images/team-photo-retina.jpg 2x, images/team-photo-full 2048w"</pre>
 
+<p>此字符串指定了在标准像素密度 (<code>1x</code>) 以及两倍像素密度 (<code>2x</code>) 下使用的图像版本。还可以使用宽度为 2048 像素 (<code>2048w</code>) 的图像版本。</p>
 <pre>"header640.png 640w, header960.png 960w, header1024.png 1024w, header.png"</pre>
 
-<p>This string provides versions of a header image to use when the {{Glossary("user agent", "user agent's")}} renderer needs an image of width 640px, 960px, or 1024px. An additional, fallback image candidate is provided without any condition at all, to be used for any other width.</p>
-
+<p>此字符串指定了在 {{Glossary("user agent", "user agent's")}} 需要渲染宽度为 640px、960px 或 1024px 的图像时使用的图像版本。以及一个额外的备用图像候选，没有任何条件，可用于任意宽度。</p>
 <pre>"icon32px.png 32w, icon64px.png 64w, icon-retina.png 2x icon-ultra.png 3x icon.svg"</pre>
 
-<p>Here, options are provided for an icon at widths of 32px and 64px, as well as at pixel densities of 2x and 3x. A fallback image is provided as an SVG file that should be used in all other cases. Notice that the candidates may use different image types.</p>
+<p>这里，为宽度为 32 像素和 64 像素以及像素密度为 2 倍和 3 倍的图标提供了选项。备用图像以 SVG 文件的形式提供，它可在所有情况下使用。请注意，图像候选可使用不同的图像类型。</p>
 
-<p>For more information on what image formats are available for use in the {{HTMLElement("img")}} element, see <a href="/en-US/docs/Web/Media/Formats/Image_types">Image file type and format guide</a>.</p>
+<p>有关可用于 {{HTMLElement("img")}} 元素的图像格式的详细信息，请参阅 <a href="/en-US/docs/Web/Media/Formats/Image_types">图片文件类型和格式指南</a> 。</p>
 
-<h2 id="Example">Example</h2>
+<h2 id="Example">例子</h2>
 
 <h3 id="HTML">HTML</h3>
 
-<p>The HTML below indicates that the default image is the 200 pixel wide version of the clock image we use in several places throughout our documentation. Also specified by the <code>srcset</code> attribute is that the 200-pixel version should be used for 1x displays while the 400-pixel version should be used for 2x displays.</p>
-
+<p>下面的 HTML 表明图像的默认宽度是 200 像素。 <code>srcset</code> 属性还指定了 200 像素版本应用于 1x 显示器，而 400 像素版本应用于 2x 显示器。</p>
 <pre class="brush: html">&lt;div class="box"&gt;
   &lt;img src="/files/16797/clock-demo-200px.png"
        alt="Clock"
```

Comment:
this image href is outdate. please async with english version.
```html
<div class="box">
  <img src="/en-us/web/html/element/img/clock-demo-200px.png"
       alt="Clock"
       srcset="/en-us/web/html/element/img/clock-demo-200px.png 1x, /en-us/web/html/element/img/clock-demo-400px.png 2x">
</div>
```

Reply:
replaced that with the prod version's absolute URL